homeassistant/components/auth/__init__.py

```diff
@@ -598,6 +601,50 @@ async def websocket_delete_refresh_token(
     connection.send_result(msg["id"], {})
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "auth/delete_all_refresh_tokens",
+    }
+)
+@websocket_api.ws_require_user()
+@websocket_api.async_response
+async def websocket_delete_all_refresh_tokens(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Handle delete all refresh tokens request."""
+
+    async def remove_safely(token: RefreshToken) -> bool:
+        # For safety, a revoke_callback could raise an error.
+        try:
+            await hass.auth.async_remove_refresh_token(token)
+            return True
+        except Exception:  # pylint: disable=broad-except
+            return False
```

Comment:
Since we're just catching `Exception` it appears as we don't expect exceptions? If so, should we log some details in stead of just quietly swallowing the error?

Reply:
Added logging entry. Is error fine here?